configs/hoefling_2024_core_readout_high_res.yaml

```
@@ -51,3 +52,5 @@ trainer:
   accumulate_grad_batches: 10
   gradient_clip_val: 1
   precision: 16-mixed
+
+only_train_readout: false # whether or not to freeze core parameters and only train readout (eg. when finetuning model to new data)
```

Maybe add an empty line at the end of the document :)

```suggestion
only_train_readout: false # whether or not to freeze core parameters and only train readout (eg. when finetuning model to new data)

```